lib/matplotlib/axes/_base.py

```diff
@@ -682,23 +682,23 @@ def get_xaxis_transform(self, which='grid'):
 
     def get_xaxis_text1_transform(self, pad_points):
         """
-        Get the transformation used for drawing x-axis labels, which
-        will add the given amount of padding (in points) between the
-        axes and the label.  The x-direction is in data coordinates
-        and the y-direction is in axis coordinates.  Returns a
-        3-tuple of the form::
-
-          (transform, valign, halign)
-
-        where *valign* and *halign* are requested alignments for the
-        text.
-
-        .. note::
+        Returns
+        -------
+        transform : Transform
+            The transform used for drawing x-axis labels, which will add
+            *pad_points* of padding (in points) between the axes and the label.
+            The x-direction is in data coordinates and the y-direction is in
+            axis corrdinates
+        valign : str
+            The text vertical alignment.
+        halign : str
```

Comment:
```suggestion
        halign : {'center', 'right', 'left'}
```

Also in subsequent copies of that docstring.

Reply:
fixed